geometry_msgs/Point r_elbow_pose
geometry_msgs/Point r_hand_pose